infra/scoped/secrets.tf

```diff
@@ -128,30 +51,113 @@ locals {
   }
 }
 
-resource "aws_secretsmanager_secret" "smoke_test-auth0_credentials" {
-  name = "identity/${terraform.workspace}/smoke_test/credentials"
+# Secrets that we set in the identity account
+module "secrets" {
+  source = "github.com/wellcomecollection/terraform-aws-secrets?ref=v1.3.0"
 
-  tags = {
-    "Name" = "identity/${terraform.workspace}/smoke_test/auth0_client_secret"
+  key_value_map = {
+    "identity/${terraform.workspace}/account_management_system/api_key" = aws_api_gateway_api_key.account_management_system.value
+    "identity/${terraform.workspace}/buildkite/credentials"             = jsonencode(local.buildkite_credentials)
+    "identity/${terraform.workspace}/smoke_test/credentials"            = jsonencode(local.smoke_test_credentials)
   }
 }
 
-resource "aws_secretsmanager_secret_version" "smoke_test-auth0_credentials" {
-  secret_id     = aws_secretsmanager_secret.smoke_test-auth0_credentials.id
-  secret_string = jsonencode(local.smoke_test_credentials)
-}
-
-# Buildkite
+# Secrets that need to be exported to the experience account
+module "secrets_experience" {
+  source = "github.com/wellcomecollection/terraform-aws-secrets?ref=v1.3.0"
 
-resource "aws_secretsmanager_secret" "buildkite-auth0_credentials" {
-  name = "identity/${terraform.workspace}/buildkite/credentials"
+  key_value_map = {
+    "identity/${terraform.workspace}/account_management_system/auth0_client_secret" = auth0_client.account_management_system.client_secret
+    "identity/${terraform.workspace}/account_management_system/api_key"             = aws_api_gateway_api_key.account_management_system.value
+  }
 
-  tags = {
-    "Name" = "identity/${terraform.workspace}/buildkite/auth0_client_secret"
+  providers = {
+    aws = aws.experience
   }
 }
 
-resource "aws_secretsmanager_secret_version" "buildkite-auth0_credentials" {
-  secret_id     = aws_secretsmanager_secret.buildkite-auth0_credentials.id
-  secret_string = jsonencode(local.buildkite_credentials)
-}
+
+// TODO: These are here to remember what state to update in prod
+// TODO: Remove these once fully applied in prod
```

Comment:
Can these be removed now?

Reply:
When I actually apply it in prod - waiting on a review first in case I need to revive anything.